src/service/authService.ts

```diff
@@ -0,0 +1,27 @@
+import type { Login } from "../model/auth";
+import axios from "axios";
+import * as dotenv from 'dotenv';
+
+export const login = async function (login: Login): Promise<string> 
+{
+    dotenv.config();
```

Comment:
It's better to load dotenv through the start script if possible 

Example 
`node -r dotenv/config dist/server.local.js"`